packages/core/src/App/Containers/Layout/header/trading-hub-header.jsx

```diff
@@ -206,46 +246,96 @@ const TradingHubHeader = ({
+                                    is_eu={is_eu}
+                                    is_virtual={is_virtual}
+                                    currency={currency}
+                                    country_standpoint={country_standpoint}
+                                    is_dialog_on={is_acc_switcher_on}
+                                    toggleDialog={toggleAccountsDialog}
+                                />
+                            </div>
+                        )}
                     </div>
                 </div>
                 <RealAccountSignup />
             </DesktopWrapper>
             <MobileWrapper>
                 <div className='trading-hub-header__mobile-parent'>
                     <div className='trading-hub-header__menu-middle'>
-                        <div className='trading-hub-header__menu-right--items--onboarding'>
-                            <TradingHubOnboarding
-                                is_dark_mode={is_dark_mode}
-                                toggleIsTourOpen={toggleIsTourOpen}
-                                is_mf={is_mf}
-                                is_eu={is_eu}
-                                is_eu_country={is_eu_country}
-                                setIsOnboardingVisited={setIsOnboardingVisited}
-                            />
-                        </div>
-                        <div className='trading-hub-header__menu-right--items--notifications'>
-                            <ShowNotifications
-                                is_notifications_visible={is_notifications_visible}
-                                notifications_count={notifications_count}
-                                toggleNotifications={toggleNotifications}
-                            />
-                        </div>
-                        <Popover
-                            classNameBubble='account-settings-toggle__tooltip'
-                            alignment='bottom'
-                            message={<Localize i18n_default_text='Manage account settings' />}
-                            should_disable_pointer_events
-                            zIndex={9999}
-                        >
-                            <BinaryLink className='trading-hub-header__setting' to={routes.personal_details}>
-                                <Icon icon='IcUserOutline' size={20} />
-                            </BinaryLink>
-                        </Popover>
-                    </div>
-                    <div className='trading-hub-header__cashier-button'>
-                        <Button primary small onClick={() => history.push(routes.cashier_deposit)}>
-                            <Localize i18n_default_text='Cashier' />
-                        </Button>
+                        {cashier_routes ? (
+                            <>
+                                <div className='trading-hub-header__menu-right--items--notifications__cashier'>
```

Comment:
please extract this login out of here to be more readable

Reply:
Already made the changes based on your suggestion 😃👍